transport/src/main/java/io/netty/channel/ChannelInitializer.java

```diff
@@ -79,6 +79,9 @@ public final void channelRegistered(ChannelHandlerContext ctx) throws Exception
             // we called initChannel(...) so we need to call now pipeline.fireChannelRegistered() to ensure we not
             // miss an event.
             ctx.pipeline().fireChannelRegistered();
+
+            // We are done with init the Channel, removing the initializer now.
```

Comment:
This comment seems to be wrong. The initializer was already removed in initChannel.